packages/itwin/map-layers/src/ui/widget/MapLayerManager.scss

```diff
@@ -268,7 +268,7 @@ $default-font-size: --iui-font-size-1;
 
     &:focus {
       outline: none;
-      box-shadow: $uicore-button-focus-boxshadow;
+      box-shadow: $buic-focus-boxshadow 0px 0px 0px 2px;
```

Comment:
Is there an itwin UI equivalent?

Reply:
@aruniverse merged jason's changes, so should be good now